### ISSUE TITLE
Ajax error: 200 parsererror

### ISSUE THREAD

**Issue by utopicchaotica** [posabsolute/jQuery-Validation-Engine#293]
I'm getting "jquery.validationEngine.js:1148 Ajax error: 200 parsererror" when I try to submit my form. I know it has to do with json malformation or something but I have no idea what's going on.

The source is here: http://utopicchaotica.net/work/AR/nstraining/application.php?appyear=2
(Sorry, I know it's a long form)

The data that logs out via console.log(data) is: enrolled=false&appFirstName=sdfdsf&appLastName=sdfdf&appAddressStreetNo=27&appAddressStreetName=sdfdf&appAddressCity=sdfsdf&appAddressProvince=ON&appAddressPostalCode=K2G4X1&appPhone=613-294-1609&appPhoneExt=1234&appOtherPhone=&appFax=&appEmail=ex%40dom.com&appDOB=1988-01-27&appBirthCert=no&appDependents=no&appDependentsExplain=&appHSGrade=12&appHSYearCompleted=1999&appHSName=sdfdf&appCourseName1=sdfdsf&appCourseType1=sdfsdf&appCourseLoc1=sdfsdf&appCourseStartDay1=2012-02-02&appCourseEndDay1=2012-03-03&appCourseCompleted1=false&appJobEmployerName1=sdfsdf&appJobLoc1=sdfsdf&appJobType1=true&appJobStartDay1=2012-02-02&appJobEndDay1=2012-03-03&appJobLeavingReason1=sdfsdf&appRefName1=sdfsdf&appRefOrg1=sdfdsf&appRefPos1=sdfsdf&appRefPhone1=5555555&appFutureWork=sdfsdf&appInterestedCourses=sdsdfsdf&appWhy2ndYear=sdfsdf&appFuturePlans=sdfsdf&appSuccessful2ndYrStudent=sdfsd

Which isn't json at all and I don't know what's going on. It should work. Any ideas?

Basically, everything validates, it just won't submit and it's tripping up in ajaxFormValidation.
